### ISSUE TITLE
Support shortened keys in URL query

### ISSUE THREAD

**Issue by justinkambic** [elastic/uptime#45]
**Is your feature request related to a problem? Please describe.**
Today, we allow the persistence of app state in the URL's query parameters like so:
![image](https://user-images.githubusercontent.com/18429259/58641530-5e154380-82c9-11e9-9704-f13d0eda843d.png)

It would be better if we could allow smaller keys so the URL doesn't get so long:
```
http://localhost:5601/app/uptime#/?autorefreshInterval=45000&autorefreshIsPaused=false&dateRangeStart=now-15m&dateRangeEnd=now&search=&selectedPingStatus=up
```

..would be shortened to:
```
http://localhost:5601/app/uptime#/?ai=45000&ap=false&ds=now-15m&de=now&q=&ps=up
```

**Describe the solution you'd like**
We will continue to support the existing keys but internally we will render the URL params with their shortened variants.

**Describe alternatives you've considered**
The alternative is doing nothing, as this feature already works.

**Additional context**
N/A

**Comment by andrewvc** [comment]
I like this idea! Agreed that it's not the highest priority ATM however.

**Comment by dov0211** [comment]
In case we would remove time picker will this still be relevant?

**Comment by justinkambic** [comment]
> In case we would remove time picker will this still be relevant?

Yes - we would still pass the URL params between pages, including _Overview_.

**Comment by andrewvc** [comment]
We could use [rison](https://github.com/Nanonid/rison) for this.
